### PR TITLE
integration bdc-auth and minor changes

### PR DESCRIPTION
**Pull request by fabianazioti**

### CODE REVIEW

File: wlts/collections/collection.py [hunk 1]
```diff
@@ -27,6 +27,7 @@ def __init__(self, name, authority_name, description, detail, datasource_id, dat
         self.scala = scala
         self.spatial_extent = spatial_extent
         self.period = period
+        self.is_public = eval(is_public)
```
Comment (raphaelrpl) [inline wlts/collections/collection.py:30]: Why do you need to use `eval` expression?

File: wlts/collections/collection_manager.py [hunk 2]
```diff
@@ -65,43 +65,43 @@ def insert(self, collection_type, collection_info):
             collection_info (dict): The collection information.
         """
         collection = CollectionFactory.make(collection_type, collection_info)
-        self._collections.append(collection)
+        self._collections[f"{collection.name}"] = collection
```
Comment (raphaelrpl) [inline wlts/collections/collection_manager.py:68]: You can simplify it passing `self._collections[collection.name] = collection`

File: wlts/collections/collection_manager.py [hunk 3]
```diff
@@ -65,43 +65,43 @@ def insert(self, collection_type, collection_info):
...
         Returns:
             list: A list with all collections identifier (name) available in the server.
         """
-        collections_names = list()
-
-        for collection in self._collections:
-            collections_names.append(collection.get_name())
-
-        return collections_names
+        if len(self._collections) > 0:
```
Comment (raphaelrpl) [inline wlts/collections/collection_manager.py:99]: When collections is empty, the function is not returning an empty list anymore. Is there any problem with that?

File: wlts/json_configs/feature_collection.json [hunk 4]
```diff
@@ -7,6 +7,7 @@
       "detail": "http://www.obt.inpe.br/OBT/assuntos/programas/amazonia/deter",
       "datasource_id": "3c20cbb4-ca94-4c1f-99af-6377f30bc683",
       "dataset_type": "Feature",
+      "is_public": "True",
```
Comment (raphaelrpl) [inline wlts/json_configs/feature_collection.json:10]: You may consider to use bool type [json.org](https://www.json.org/json-en.html#text)

**Review by raphaelrpl — CHANGES_REQUESTED** [review]

File: wlts/collections/collection.py (hunk 1 above)
Comment (fabianazioti) [reply wlts/collections/collection.py]: It is not necessary! I switched to the boolean type in json. Thank you! @raphaelrpl

File: wlts/json_configs/feature_collection.json (hunk 4 above)
Comment (fabianazioti) [reply wlts/json_configs/feature_collection.json]: Done, thank you!

File: wlts/collections/collection_manager.py (hunk 3 above)
Comment (fabianazioti) [reply wlts/collections/collection_manager.py]: Must return empty list

**Review by raphaelrpl — APPROVED** [review]

**Pull request merged**